kolibri/plugins/learn/assets/src/vue/search-widget/search-card.vue

```diff
@@ -2,155 +2,8 @@
-                    <path d="M2,15 C2,7.83375 7.83375,2 15,2 C22.16625,2 28,7.83375 28,15 C28,22.16625 22.16625,28 15,28 C7.83375,28 2,22.16625 2,15 Z M15,4 C8.928,4 4,8.928 4,15 C4,21.072 8.928,26 15,26 C21.072,26 26,21.072 26,15 C26,8.928 21.072,4 15,4 Z" fill="#996189"></path>
-                    <polygon fill="#996192" points="15 18.689 19.326 21.3 18.178 16.379 22 13.068 16.967 12.641 15 8 13.033 12.641 8 13.068 11.822 16.379 10.674 21.3"></polygon>
-                </g>
-            </g>
-        </svg>
-      </div>
-
-      <div class="content-icon" v-if="kind === 'exercise' && progress === 'partial'">
-        <svg width="30px" height="30px" viewbox="0 0 30 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-                <g>
-                    <path d="M15,30 C23.2842712,30 30,23.2842712 30,15 C30,6.71572875 23.2842712,3.55271368e-15 15,3.55271368e-15 C6.71572875,3.55271368e-15 0,6.71572875 0,15 C0,23.2842712 6.71572875,30 15,30 Z" fill="#FFFFFF"></path>
-                    <path d="M4,15 C4,21.072 8.928,26 15,26 C21.072,26 26,21.072 26,15 L4,15 Z" fill="#E2D1E0"></path>
-                    <path d="M2,15 C2,7.83375 7.83375,2 15,2 C22.16625,2 28,7.83375 28,15 C28,22.16625 22.16625,28 15,28 C7.83375,28 2,22.16625 2,15 Z M15,4 C8.928,4 4,8.928 4,15 C4,21.072 8.928,26 15,26 C21.072,26 26,21.072 26,15 C26,8.928 21.072,4 15,4 Z M15,18.689 L19.326,21.3 L18.178,16.379 L22,13.068 L16.967,12.641 L15,8 L13.033,12.641 L8,13.068 L11.822,16.379 L10.674,21.3 L15,18.689 Z" fill="#996192"></path>
-                </g>
-            </g>
-        </svg>
-      </div>
-
-      <div class="content-icon" v-if="kind === 'exercise' && progress === 'complete'">
-        <svg width="30px" height="30px" viewbox="0 0 30 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-                <g>
-                    <path d="M15,30 C23.2842712,30 30,23.2842712 30,15 C30,6.71572875 23.2842712,3.55271368e-15 15,3.55271368e-15 C6.71572875,3.55271368e-15 0,6.71572875 0,15 C0,23.2842712 6.71572875,30 15,30 Z" fill="#FFFFFF"></path>
-                    <path d="M2,15 C2,7.83375 7.83375,2 15,2 C22.16625,2 28,7.83375 28,15 C28,22.16625 22.16625,28 15,28 C7.83375,28 2,22.16625 2,15 Z M15,18.689 L19.326,21.3 L18.178,16.379 L22,13.068 L16.967,12.641 L15,8 L13.033,12.641 L8,13.068 L11.822,16.379 L10.674,21.3 L15,18.689 Z" fill="#996192"></path>
-                </g>
-            </g>
-        </svg>
-      </div>
-
-
+      <img class="content-icon" v-if="kind === 'topic'" src="../folder.svg">
```

Comment:
That would be possible too. I decided against because topics don't have the
same set of completion states as contents
On Jul 11, 2016 4:20 PM, "Eli" <notifications@github.com> wrote:

> In kolibri/plugins/learn/assets/src/vue/search-widget/search-card.vue
> <https://github.com/66eli77/kolibri/pull/2#discussion_r70354630>:
>
> > -        </svg>
> > -      </div>
> > -
> > -      <div class="content-icon" v-if="kind === 'exercise' && progress === 'complete'">
> > -        <svg width="30px" height="30px" viewbox="0 0 30 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
> > -            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
> > -                <g>
> > -                    <path d="M15,30 C23.2842712,30 30,23.2842712 30,15 C30,6.71572875 23.2842712,3.55271368e-15 15,3.55271368e-15 C6.71572875,3.55271368e-15 0,6.71572875 0,15 C0,23.2842712 6.71572875,30 15,30 Z" fill="#FFFFFF"></path>
> > -                    <path d="M2,15 C2,7.83375 7.83375,2 15,2 C22.16625,2 28,7.83375 28,15 C28,22.16625 22.16625,28 15,28 C7.83375,28 2,22.16625 2,15 Z M15,18.689 L19.326,21.3 L18.178,16.379 L22,13.068 L16.967,12.641 L15,8 L13.033,12.641 L8,13.068 L11.822,16.379 L10.674,21.3 L15,18.689 Z" fill="#996192"></path>
> > -                </g>
> > -            </g>
> > -        </svg>
> > -      </div>
> > -
> > -
> > +      <img class="content-icon" v-if="kind === 'topic'" src="../folder.svg">
>
> why not also put folder.svg inside content-icon component?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/66eli77/kolibri/pull/2/files/c720088f238e7fd96ed221cb23783a99ed42605d#r70354630>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACQfIfbp2KqfjgTmNSiyQdNKlI_Xl_FBks5qUs_JgaJpZM4JJ498>
> .
>